app/templates/events/view/sessions.hbs

```diff
@@ -0,0 +1,146 @@
+<div>
+  <div class="left-float">
+    <h2> Event Name  </h2>
+  </div>
+  <div class="left-float">
+    <h2> | </h2>
+  </div>
+  <div class="left-float">
+    <h2>  Sessions </h2>
+  </div>
+  <div class="right-float">
+    <button class="ui button"> Preview </button>
+    <button class="ui button"> Publish </button>
+    <button class="ui button"> Copy </button>
+    <button class="ui red button"> Delete </button>
+  </div>
+</div>
+<br>
+<br>
+<br>
+<div>
+  <div class="ui buttons">
+    <button class="ui button">Overview</button>
+    <button class="ui button">Tickets</button>
+    <button class="ui button">Scheduler</button>
+    <button class="ui active button">Sessions</button>
+    <button class="ui button">Speakers</button>
+    <button class="ui button">Export</button>
+  </div>
+  <div class="ui divider"></div>
+  <br>
+  <div class="ui buttons">
+    <button class="ui button">All</button>
+    <button class="ui button">Pending</button>
+    <button class="ui button">Accepted</button>
+    <button class="ui button">Confirmed</button>
+    <button class="ui button">Rejected</button>
+  </div>
+  <button class="ui blue button">Create Session</button>
+  <button class="ui blue button">Export as CSV</button>
+  <div class="search-box">
+    <div class="ui icon input">
+      <input class="prompt" type="text" placeholder="Search">
+      <i class="search icon"></i>
+    </div>
+    <div class="results"></div>
+  </div>
+  <br>
+  <br>
+  <div class="show enteries">
+    <div class="text-show"> Show </div>
+    <div class="left-float">
+      <select class="ui dropdown">
+        <option value="0">10</option>
+        <option value="1">20</option>
+        <option value="2">50</option>
+        <option value="3">100</option>
+      </select>
+    </div>
+    <div class="text-enteries"> Enteries </div>
+  </div>
+  <br>
+  <br>
+</div>
+<div class="row">
+  <div class="sixteen wide column">
+    <table class="ui fixed table">
+      <thead>
+        <tr>
+          <th>{{t 'State'}}</th>
+          <th>{{t 'Title'}}</th>
+          <th>{{t 'Speakers'}}</th>
+          <th>{{t 'Track'}}</th>
+          <th>{{t 'Short Abstract'}}</th>
+          <th>{{t 'Submission Date'}}</th>
+          <th>{{t 'Last Modified'}}</th>
+          <th>{{t 'Email Sent'}}</th>
+          <th></th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <div>
+              <button class="ui green button">Confirmed</button>
```

Comment:
Why are you using a button when there is are labels for just this purpose ?

see -> https://semantic-ui.com/elements/label.html